forge/postoffice/templates/TeamInvitation.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
 
 You've been invited to join team {{{invite.team.name}}} on the FlowForge platform.
 
-{{{ signupLink }}}
+{{{ signupLink }}}/account/teams/invitations
```

Comment:
We shouldn't have this partial url in the template. It should be set entirely via `signupLink`.

Reply:
sorry missed removing it on the plain text template, fix now